src/osgEarthDrivers/label_mp_screen_annotations/MPAnnotationLabelSource.cpp

```diff
@@ -135,6 +137,9 @@ class MPAnnotationLabelSource : public LabelSource
 
                 if ( text->autoOffsetGeomWKT().isSet() )
                     tempStyle.get<TextSymbol>()->autoOffsetGeomWKT()->setLiteral( feature->eval( textOffsetSupportExpr, &context ) );
+                
+                if ( text->textPriority().isSet() )
+                    tempStyle.get<TextSymbol>()->textPriority()->setLiteral( feature->eval( textPriorityFineExpr, &context ) );
```

Comment:
**Impossible to understand the difference between TextSymbol::priority() and TextSymbol::textPriority**

replace textPriority by priorityFine everywhere